templates/systems/element_detail_tabs.html

```diff
@@ -87,176 +87,204 @@
 {% block contextbar %}{% endblock %}
 
 {% block body %}
+    <div class="systems-top">
+      <!-- <div class="container"> -->
+      <div class="row">
+          <div id="above-tab-content" class="row">
+              <!-- Page Title-->
+              <div class="systems-title">
+                  <h2 class="control-heading">
+                      <a href="{{ project.get_absolute_url }}" class="systems-link">{{ system.root_element.name }}</a> &gt; <a href="/systems/{{system.id}}/components/selected" class="systems-link">Selected Components</a>
+                  </h2>
+              </div>
+              <!-- Control Lookup-->
+              <div class="systems-control-lookup">
+                  <!-- <form id="control-lookup" method="get" action="/controls/" onsubmit="show_control_by_id(); return false;">
```

Comment:
This section is also commented out in element_detail. Not sure if it is useful to keep.

Reply:
Controls search form has been commented out.

Reply:
I meant to say it should just be removed then if it is not used